src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
 // write your code here
+const toClone = document.querySelector('table');
```

Comment:
```suggestion
const table = document.querySelector('table');
```
If you store the table in the variable the name should be corresponding